### PR TITLE
[16.0][MIG] maintenance_request_sequence

### PR DESCRIPTION
**Pull request by christian-ramos-tecnativa**
https://github.com/OCA/maintenance/issues/240

### CODE REVIEW

**Review by szalatyzuzanna — APPROVED** [review]

File: maintenance_request_sequence/views/maintenance_team.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 Creu Blanca
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="maintenance_team_view_form">
```
Comment (dalonsod) [inline maintenance_request_sequence/views/maintenance_team.xml:6]: Is it possible to add `code_prefix` field to team `tree` column? For now, the only way to edit this value for existing during team creation or modification is first switching to `kanban` view.

**Review by dalonsod — CHANGES_REQUESTED** [review]
Tested on runboat 👍 . See comment for approval.

File: maintenance_request_sequence/views/maintenance_team.xml (hunk 1 above)
Comment (christian-ramos-tecnativa) [reply maintenance_request_sequence/views/maintenance_team.xml:6]: Added and thanks for the review

**Review by dalonsod — APPROVED** [review]

**Review by etobella — APPROVED** [review]

**Pull request merged**